homeassistant/components/metoffice/strings.json

```diff
@@ -5,7 +5,7 @@
         "description": "The latitude and longitude will be used to find the closest weather station.",
         "title": "Connect to the UK Met Office",
         "data": {
-          "api_key": "Met Office DataPoint API key",
+          "api_key": "[%key:common::config_flow::data::api_key%]",
```

Comment:
this string might be too generic, no ?

Reply:
The unique info to this integration could be placed in the title or description instead?  The only extra info the existing string provides is DataPoint

Reply:
Or use of partial reference key : "`Datapoint [%key:common::config_flow::data::api_key%]`"

Reply:
The only api key that Met Office provides is a "Datapoint api key". I think that we're fine in dropping it.

Reply:
Being picky, MetOffice has two API keys now - DataPoint and DataHub - but there's no obvious PyPi-based way of accessing the latter, so it's unlikely that we'll be shifting over to that one in the near future.

Happy with the generic message, for the time being.